src/main/resources/reference.conf

```diff
@@ -774,17 +777,20 @@ resourceTypes = {
       link = {
         description = "link a billing-project to a google-project"
       }
+      read = {
+        description = "read this billing-project"
+      }
```

Comment:
`billing-project` has a `view-status` action … are these equivalent?

Reply:
Removed this based on standup parking lot: https://github.com/broadinstitute/sam/pull/1656/commits/3992ee4e68ac577758f4c6cd920fe07362deb7fe